backend/eventloop/db_reconciler_test.go

```diff
@@ -451,7 +451,17 @@ var _ = Describe("DB Reconciler Test", func() {
 				err = dbq.CreateAPICRToDatabaseMapping(ctx, &apiCRToDatabaseMappingDb)
 				Expect(err).To(BeNil())
 
-				_, _, _, gitopsEngineInstance, _, err := db.CreateSampleData(dbq)
+				dbQueries, err := db.NewUnsafePostgresDBQueries(true, true)
+				Expect(err).To(BeNil())
+				_, _, engineCluster, _, _, err := db.CreateSampleData(dbQueries)
+				Expect(err).To(BeNil())
+				gitopsEngineInstance := &db.GitopsEngineInstance{
+					Gitopsengineinstance_id: "test-fake-instance-id",
+					Namespace_name:          "gitops-service-argocd",
+					Namespace_uid:           "test-fake-instance-namespace-914",
+					EngineCluster_id:        engineCluster.Gitopsenginecluster_id,
+				}
+				err = dbQueries.CreateGitopsEngineInstance(ctx, gitopsEngineInstance)
```

Comment:
gitopsEngineInstance here needs to be in the hard coded namespace : gitops-service-argocd ([here](https://github.com/redhat-appstudio/managed-gitops/blob/main/backend/eventloop/application_event_loop/application_event_runner_deployments.go#L184)) or else the test would fail that is why not using the generic function.